### PR TITLE
Drop Ubuntu 14.04 & add Debian 9/10 / Fedora 29/30 support

### PR DESCRIPTION
**Pull request by ekohl**
Adds Debian 9, Debian 10 and Fedora 30. Drops Debian 8 and Ubuntu 14.04.

### CODE REVIEW

**Comment by bastelfreak** [comment]
Personally, I would be fine with keeping operating systems that still have upstream support in the metadata.json file.

**Comment by ekohl** [comment]
When Debian 10 was released, the Debian 8 backports repositories were dropped. https://backports.debian.org/ states:
> Please keep in mind that backports doesn't follow LTS. Which means that we will drop support for oldstable (stretch) around one year after the release of buster. Thats in sync with the - official - security support for oldstable

We could set the type to VCS for that, but I'd prefer not to and consider it EOL.

**Comment by bastelfreak** [comment]
@ekohl okay sounds good

File: README.md [hunk 1]
```diff
@@ -171,7 +156,7 @@ Example:
 
 ```puppet
 class { 'letsencrypt::plugin::dns_rfc2136':
-  server     => '1.2.3.4',
+  server     => '192.0.2.1',
```
Comment (ekohl) [inline README.md]: This is TEST-NET-1 (192.0.2.0/24) from [rfc5737](https://datatracker.ietf.org/doc/rfc5737/).

**Pull request merged**